docs/internals/release-process.txt

```diff
@@ -71,6 +53,54 @@ security purposes, please see :doc:`our security policies <security>`.
     So the answer to "should I upgrade to the latest minor release?" will always
     be "yes."
 
+Starting with Django 3.0, version numbers will use a loose form of `semantic
+versioning <http://semver.org/>`_. SemVer makes it easier to see at a glance
+how compatible releases are with each other. It also helps to anticipate when
+compatibility shims will be removed. It’s not a pure form of SemVer as each
+feature release will continue to have some documented backwards
```

Comment:
I'd prefer to switch from `some` to `a few` here. How often do we actually do this, when it comes to documented APIs?